src/applications/disability-benefits/all-claims/content/toxicExposure.jsx

```diff
@@ -42,3 +56,100 @@ export const conditionsDescription = (
 export const gulfWar1990PageTitle = 'Service locations after August 2, 1990';
 export const gulfWar1990Question =
   'Did you serve in any of these Gulf War locations on or after August 2, 1990? Check any locations where you served.';
+
+/**
+ * Builds the Schema based on user entered condition names
+ * 
+ * Example output:
+{
+    type: 'object',
+    properties: {
+      anemia: {
+        type: 'boolean'
+      },
+      tinnitusringingorhissinginears: {
+        type: 'boolean'
+      },
+      none: {
+        type: 'boolean'
+      }
+    }
+  }
+}
+ *
+ * @param {object} formData - Full formData for the form
+ * @returns {object} Object with id's for each condition
+ */
+export const makeTEConditionsSchema = formData => {
+  const options = (formData?.newDisabilities || []).map(disability =>
+    sippableId(disability.condition),
+  );
+
+  options.push('none');
+
+  return checkboxGroupSchema(options);
+};
+
+/**
+ * Builds the UI Schema based on user entered condition names.
+ *
+ * Example output:
+ *  {
+ *   anemia: {
+ *     'ui:title': 'Anemia',
+ *   },
+ *   tinnitusringingorhissinginears: {
+ *     'ui:title': 'Tinnitus (Ringing Or Hissing In Ears)',
+ *   },
+ *   none: {
+ *     'ui:title': 'I am not claiming any conditions related to toxic exposure',
+ *   },
+ * }
+ * @param {*} formData - Full formData for the form
+ * @returns {object} Object with id and title for each condition
+ */
+export const makeTEConditionsUISchema = formData => {
+  const { newDisabilities = [] } = formData;
+  const options = {};
+
+  newDisabilities.forEach(disability => {
+    const { condition } = disability;
+
+    const capitalizedDisabilityName =
+      typeof condition === 'string'
+        ? capitalizeEachWord(condition)
+        : NULL_CONDITION_STRING;
+
+    options[sippableId(condition || NULL_CONDITION_STRING)] = {
+      'ui:title': capitalizedDisabilityName,
+    };
+  });
+
+  options.none = {
+    'ui:title': 'I am not claiming any conditions related to toxic exposure',
+  };
+
+  return options;
+};
+
+export const noneAndConditionError =
+  'You selected a condition, and you also selected “I’m not claiming any conditions related to toxic exposure.” You’ll need to uncheck one of these options to continue.';
```

Comment:
Just a clarity consideration--could "Uncheck one of these options to continue" (without the "You'll need to" part) be more clear?

Reply:
This content had some back and forth with Mike and the CAIA team and this was the finalized text. I imagine it was constructed in this way to use [active voice](https://design.va.gov/content-style-guide/plain-language/use-active-voice). I think if we have feedback we should check if there was a ticket for that effort or we can check with Mike when he is back.